pkg/app/application.go

```diff
@@ -160,12 +157,19 @@ func (a *Application) loadCustomizedResMap() (resmap.ResMap, error) {
 		errs.Append(errors.Wrap(err, "NewResourceSliceFromPatches"))
```

Comment:
Please move L154 into a function called, say, `handleAliasedAndDeprecatedFields`, and call it at the top.